src/components/icon-dropdown/index.js

```diff
@@ -74,7 +77,7 @@ class Dropdown extends Component {
                     />
                 {visible &&
                     <div data-focus='dropdown-menu' data-position={openDirection} ref='dropdown'>
-                        {operationList.map(({label, action}, idx) => (<div key={idx} data-role='dropdown-item' onClick={this._operationActionWrapper(action)}>{label}</div>))}
+                        {operationList.map(({label, action}, idx) => (<div key={idx} data-role='dropdown-item' onClick={this._operationActionWrapper(action)}>{this.i18n.t(label)}</div>))}
```

Comment:
Have you tested it ? I doubt about the `this.i18n.t`, I think you meant `this.i18n`.